README.md

```diff
@@ -45,34 +41,50 @@ export const runQuery = makeSafeQueryRunner(query => client.fetch(query));
 
 // 👇 Now you can run queries and `data` is strongly-typed, and runtime-validated.
 const data = await runQuery(
-  q(
-    "*",
-    q.filter("_type == 'pokemon'"),
-    q.grab({ name: q.string() }),
-    q.slice(0, 150),
-  )
+  q("*")
+    .filter("_type == 'pokemon'")
+    .grab({ name: q.string() })
+    .slice(0, 150)
 );
 // data: { name: string }[]
 ```
 
 Using `makeSafeQueryRunner` is totally optional; you might find using `q().schema` and `q().query` in your own abstractions works better for you.
 
-## Why? 🤷‍
+## **NOTE**: Significant API changes with 0.3.0
```

Comment:
Ugh, I'm torn on this. These are breaking changes, and should probably be released as a major version change. Maybe I should be planning to release as a 1.0.0-alpha.

Reply:
In sub-v1.0.0 semantic versioning, 0.3.0 is appropriate for breaking changes.

<img width="519" alt="image" src="https://user-images.githubusercontent.com/4413963/212117352-0e2f4626-64ec-4088-bc7a-2a0baff39e32.png">
https://semver.npmjs.com/